### PR TITLE
Backport from MetaDeploy.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Based on https://github.com/SFDO-Tooling/metadeploy/compare/e41ac55...6b5b3162ab5f1e4e3908c4fbdb6cd8fcf9aecd18

whale

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+def fernet_encrypt(s):
+    """Encrypt a string using cryptography.fernet"""
+    return FERNET.encrypt(s.encode("utf-8")).decode("utf-8")
+
+
+def fernet_decrypt(s):
+    """Decrypt a string using cryptography.fernet"""
+    return FERNET.decrypt(s.encode("utf-8")).decode("utf-8")
+
+
```
Comment (davisagli) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py]: We should move everything in this file to sfdo-template-helpers.

**Review by davisagli — APPROVED** [review]
LGTM, aside from noticing another thing to move to the helpers repo.

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py (hunk 1 above)
Comment (jgerigmeyer) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py]: Makes sense. @wlonk You want to start by abstracting that and making the relevant changes in MD, then we can backport over here?

Comment (wlonk) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py]: @jgerigmeyer Done. I've added a commit here to import it from the correct spot.

**Pull request merged**